core/src/test/java/org/elasticsearch/index/mapper/DynamicMappingDisabledIT.java

```diff
@@ -0,0 +1,29 @@
+package org.elasticsearch.index.mapper;
```

Comment:
If this [does end up needing to be an integration test](https://github.com/elastic/elasticsearch/pull/15424/files#r47975786), this file will need a license header.